### ISSUE TITLE
unable to install 0.9.x

### ISSUE THREAD

**Issue by stephenmathieson** [duojs/duo#474]
i've got an app using duo `0.9` (and it's stuck there until duojs/pack#43 is resolved).  unfortunately, i'm no longer able to install `duo@0.9` due to some crazy npm nonsense:

```
$ npm install duo@0.9
npm ERR! addLocal Could not install /var/folders/g3/3swz_5rx5qv8z15sq9d3wcqm0000gn/T/npm-21371-70665ef0/git-cache-92a32f840518/3bce55b6546f31ce0c62f4cf81f041e9e8606e42
npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/stephen/.nvm/versions/io.js/v2.2.1/bin/iojs" "/Users/stephen/.nvm/versions/io.js/v2.2.1/bin/npm" "install" "duo@0.9"
npm ERR! node v2.2.1
npm ERR! npm  v2.11.3
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read

npm ERR! EISDIR: illegal operation on a directory, read
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/stephen/npm-debug.log
```

i've tracked the issue down to our `gnode@0.1.0` dep.  would anyone be opposed to me releasing a `0.9.5` with `gnode@0.1.1`?

this is blocking my team currently, so immediate input would be greatly appreciated.

/cc @dominicbarnes @matthewmueller @yields

**Comment by dominicbarnes** [comment]
No objection here, go right ahead!

**Comment by stephenmathieson** [comment]
wtf there's already a `0.9.5` and a `0.9.6` in npm ... there's just no git tag anywhere

**Comment by stephenmathieson** [comment]
released `0.9.7`

**Comment by dominicbarnes** [comment]
How weird, does npm show who published those versions?
